README.md

```diff
@@ -1,3 +1,18 @@
 # Literature Reviews
 
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review. You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request.
+
+## The process
+
+* Fork the https://github.com/elm-community/literature-reviews
```

Comment:
How about `Fork [the elm-community/literature-reviews repo](https://github.com/elm-community/literature-reviews)`?